downstream/assemblies/platform/assembly-planning-installation.adoc

```diff
@@ -13,10 +13,14 @@ ifdef::context[:parent-context: {context}]
 You can use this section to help plan your {PlatformName} installation. Before installation, review information on the setup installer, system requirements, and supported installation scenarios.
```

Comment:
If possible, could you erase the self reference in the intro, to something like this maybe: "To help plan your {PlatformName} installation, review information on the setup installer, system requirements, and supported installation scenarios."